File: Euclid's Algorithms/gcd/gcd.cpp
#include<bits/stdc++.h>
using namespace std;

int gcd(int a, int b) {
	if(a == 0)
		return b;
	return gcd(b % a, a);
}

int main() {
	int a, b;
	cout << "Enter a, b -\n";
	cin >> a >> b;
	int res = gcd(a, b);
	cout << "gcd(a, b) = " << res;
	cout << "\n";
}

File: Euclid's Algorithms/gcd/gcd.py
def gcd(a, b):
	if a == 0:
		return b
	return gcd(b % a, a)


a = int(input("a = "))
b = int(input("b = "))
res = gcd(max(a, b), min(a, b))
#print(res)
print("gcd(a, b) = ", res)
